AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -277,6 +277,7 @@ - (void)_initializeInstance
   _contentsScaleForDisplay = ASScreenScale();
   _displaySentinel = [[ASSentinel alloc] init];
   _preferredFrameSize = CGSizeZero;
+  _pendingViewState = [_ASPendingState new];
```

Comment:
Is this necessary? I have definitely seen overhead from the pending state object in traces, and there are nodes that don't end up using it at all.

Assuming it is necessary though, we should use the standard alloc / init simply because the new method is not used almost anywhere else in the framework.